### PR TITLE
Warning messages of validator

### PR DESCRIPTION
**Pull request by sandertan**
# What? Why?
Small changes to the validator.

Changes proposed in this pull request:
- Warning messages are now more clear about what might be wrong with the gene
symbol or Entrez gene when they are not in the database.
- Fixed some text to be more consistent on the used terms for gene symbol and Entrez gene.

### CODE REVIEW

**Comment by sandertan** [comment]
Also added small fix to 'help' message of metaImport regarding JAR path. @fedde-s

File: core/src/main/scripts/importer/metaImport.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def interface():
                                        help='Skip tests requiring information '
                                             'from the cBioPortal installation')
     parser.add_argument('-jar', '--jar_path', type=str, required=False,
-                        help='Path to core JAR file')
+                        help='Path to scripts JAR file')
```
Comment (fedde-s) [inline core/src/main/scripts/importer/metaImport.py]: Great, this fixes the issue I ran into, by documenting that this argument was affected by c594d70676fe136fba85ae8172307ef53a3353ac as well.

File: core/src/main/scripts/importer/validateData.py [hunk 2]
```diff
@@ -594,24 +594,24 @@ def checkGeneIdentification(self, gene_symbol=None, entrez_id=None):
         """Attempt to resolve a symbol-Entrez pair, logging any issues.
 
         It will fail to resolve in these cases:
-            1. (error) Entrez id and symbol are both missing (None)
+            1. (error) Entrez gene and gene symbol are both missing (None)
```
Comment (pieterlukasse) [inline core/src/main/scripts/importer/validateData.py]: "Entrez gene id" is better I think

File: core/src/main/scripts/importer/validateData.py [hunk 3]
```diff
@@ -661,29 +661,32 @@ def checkGeneIdentification(self, gene_symbol=None, entrez_id=None):
...
+                            'wrong mapping or recycled gene symbol.',
                             extra={'line_number': self.line_number,
                                    'cause': '(%s, %s)' % (gene_symbol,
                                                           entrez_id)})
             else:
                 self.logger.warning(
-                    'Entrez gene id not known to the cBioPortal instance. '
-                    'This gene will not be loaded',
+                    'Entrez gene not known to the cBioPortal instance. '
+                    'This gene will not be loaded. Might be new or deprecated '
```
Comment (pieterlukasse) [inline core/src/main/scripts/importer/validateData.py]: maybe even clearer if it states that "This record will not be loaded" or "This record will be skipped"?

File: core/src/main/scripts/importer/validateData.py [hunk 4]
```diff
@@ -1009,8 +1012,8 @@ def checkHeader(self, cols):
                 extra={'line_number': self.line_number})
         elif not 'swissprot_identifier' in self.meta_dict:
             self.logger.warning(
-                "A SWISSPROT column was found in a file without an "
-                "associated 'swissprot_identifier' metadatum, assuming "
+                "A SWISSPROT column was found without an associated "
+                "'swissprot_identifier' attribute; assuming "
```
Comment (pieterlukasse) [inline core/src/main/scripts/importer/validateData.py]: "...attribute in the mutation meta file".... i.e. just using another word will not make it clearer. Add some more info.

**Comment by sandertan** [comment]
Updated to Pieter's remarks.

**Pull request merged**